configs.py

```diff
@@ -22,3 +22,5 @@ class Config:
     SQLALCHEMY_TRACK_MODIFICATIONS = False
     SQLALCHEMY_DATABASE_URI = os.environ.get('SQLALCHEMY_DATABASE_URI') or alternative_db
     SQL_LITE_DEFAULT = False or (alternative_db != None)
+    RESOURCES_PER_PAGE = 20
```

Comment:
1500/20 = 75 pages.
50?/20 = 2.5 pages. 

![bigger default](https://66.media.tumblr.com/tumblr_lxsxbyWVL61qj5ps7o1_400.gif "Going to need a bigger default" )

Reply:
I chose somewhat random values. I'm open to suggestions on better defaults.